### DISCUSSION TITLE
Hope this project isn't dead

### DISCUSSION THREAD

**Discussion by JPcode05** [tenacityteam/tenacity-legacy#677] [General]
This project hasn't made a commit in almost a month and the staff haven't been active in the issues and PR's except for [this one patch on the mailing list](https://lists.sr.ht/~tenacity/tenacity-dev/patches/26071).

What's happened to this project? I have hopes for success of this project in the future.

**Comment by poyribn** [comment]
I also hope this.

**Reply by FrostKnight** [reply]
IF it does die, I recommend it gets forked asap.

Although, I like that the lead dev of this project, the current one had the method of making this changed to use cmake and that it no longer requires that bloat conan. ;)

I hope it doesn't die either though, just so you all know.

Anything is better than than one fork of audacityt that was ran by people who harrased people on here... except audacity itself of course which is also bad.

-_-

**Reply by n0toose** [reply]
We'll continue to review patches and are willing to (eventually) integrate anyone who's willing to take on a lot of the heavy lifting into our team and eventually give write access, by the way.

**Reply by n0toose** [reply]
See https://github.com/tenacityteam/tenacity/discussions/677#discussioncomment-1671963

**Comment by Be-ing** [comment]
We're [not dead yet](https://www.youtube.com/watch?v=Jdf5EXo6I68&t=56s)! Just put Tenacity stuff on the backburner for a bit...

**Reply by n0toose** [reply]
I'm personally busy with dealing with a very hard university, another person had IRL issues, some contributors that contributed features lost the available free time they had for now...

It's just that all of us are held up with other things for the time being. I'll definitely make some time to review patches if anyone sends in some.

**Reply by caughtquick** [reply]
On my side of things I'm still in high schools and I just have a lot on my plate with finals week coming up, this week has really been the only time for me to relax a bit since it started and don't really want to spend that time on tenacity (no offense)

**Reply by JPcode05** [reply]
> On my side of things I'm still in high schools and I just have a lot on my plate with finals week coming up, this week has really been the only time for me to relax a bit since it started and don't really want to spend that time on tenacity (no offense)

I feel you. I just finished my finals at high school. And since there was no school during the summer, it was a lot easier to start on this project during that time, as it slowed down when school started back up.

But we'll still accept patches from those who have anything.

**Reply by FrostKnight** [reply]
I think that in general it would be wise, if  developers or contributors are available, etc...

https://github.com/saucedacity/saucedacity

work on the above project, this person is still working on it since when he started.

:)

That being said, working on both would be wise in general.

Or just the other one, if there is no other way...

I will link this in case someone has interest at the bottom of the page if possible though.

**Comment by RockyMM** [comment]
Working on an audio editor is hard. Working on the Audacity fork is even harder. I really think that forking Audacity was not the best idea. The community would have had much more value out of joining efforts with the core Audacity team and resolving any and all issues with them. You simply cannot compare with developers which maintain a piece of software as a day job compared to devs who do it in their own free time.

**Reply by Be-ing** [reply]
Trying to work together with a company that repeatedly acts in bad faith is pointless.

**Reply by Semisol** [reply]
> Trying to work together with a company that repeatedly acts in bad faith is pointless.

They were bought out by Muse Group, then added telemetry the community opposed to, then requested everyone sign a CLA (this was said to be for relicensing, but I believe it is for their own commercial gain, and also I believe they kept some non-CLA code).

**Reply by RockyMM** [reply]
> > Trying to work together with a company that repeatedly acts in bad faith is pointless.
> 
> They were bought out by Muse Group, then added telemetry the community opposed to, then requested everyone sign a CLA (this was said to be for relicensing, but I believe it is for their own commercial gain, and also I believe they kept some non-CLA code).

If any of the OG devs wanted to leave - they could have left. To me it seems like a matter of opinion; not matter of fact. The community mostly stayed with the upstream, or so it seems to me?

**Reply by Semisol** [reply]
> If any of the OG devs wanted to leave - they could have left. To me it seems like a matter of opinion; not matter of fact. The community mostly stayed with the upstream, or so it seems to me?

I believe the OG devs are contracted by Muse Group too, as they are the only one that could have "transferred control".

**Reply by RockyMM** [reply]
Anyways, let’s revisit the project in few months time and see if the effort spent was well spent or it could have been spent better elsewhere.

**Reply by Semisol** [reply]
> Anyways, let’s revisit the project in few months time and see if the effort spent was well spent or it could have been spent better elsewhere.

I can tell you that not letting Muse steal our code with a CLA was a good idea.

**Reply by Paul-Licameli** [reply]
Hello from upstream.

I appreciate @RockyMM ’s call for amity.

I have said without sarcasm that I regret that the dedicated people at Tenacity can’t combine efforts with us to make Audacity a better free, open source program, ultimately to benefit its users, which is the most important thing.

And it has remained a program freely downloadable, with source code anyone may obtain, and study, and learn from, and reuse.  It has released version 3.1.0, still under such terms, and with some enhancements.

**Reply by mc776** [reply]
Any "call for 'amity'", if not accompanied by an immediate, irreversible and ongoing structural change to stop and prevent the underlying problems that required the fork, is at best a hollow beg to come back to an abuser, at worst a continuation of that abuse.

**Reply by RockyMM** [reply]
Well, hello from the future

**Comment by aeiouaeiouaeiouaeiouaeiouaeiou** [comment]
Still hoping for a stable release, maybe we should expect something special before the end of the year? 🤔

**Reply by mc776** [reply]
44 current issues marked "bug", including one marked "high priority" and a couple crashes.

I know that people involved have been really busy since summer ended and this Christmas holiday's been a clusterfuck for everybody.

Maybe just let people talk for now - lots of FOSS projects stall or have janky latest versions for much longer than this.

**Reply by Be-ing** [reply]
I'm willing to invest a bit of effort to revive this if I'm not doing it alone, but I'm largely out of patience for anything C or C++ at this point and spending my time on Rust now.

**Reply by FrostKnight** [reply]
I must admit, I don't really understand why everyone wants to rustify every package... it puzzles me to no end...

Especially given the trademark issues it has...

https://github.com/rust-lang/rust/issues/77234
https://github.com/rust-lang/rust/issues/53287

Also, it seems to be one of the largest programming languages out there...

**Comment by FrostKnight** [comment]
https://github.com/tenacityteam/tenacity/discussions/729

At this point, I think it would be wise to heed what I posted here, until he has time or in general, or both.

This fork is still being worked on since its creation:

https://github.com/saucedacity/saucedacity

Please do so, otherwise, audacity will still hold the reigns as being something that it used to be, despite its treachery...

Long story short, don't surrender to audacity or give up.

This statement hopefully will be well received...

;)

**Reply by generic-pers0n** [reply]
Hello! Lead Saucedacity dev here! Just wanna say a couple of things that I hope welcomes you guys to Saucedacity if you feel like coming over, but before that, one thing: yes, the name is not the best, so I'm not gonna say I come up with the best project names or anything. And you don't have to come over if you don't want to either way :)

Indeed I have been working on this for some year now, so we're still going. However, after a year of working on Saucedacity myself, I'm ready to welcome anyone who's willing to contribute. With that being said, you guys are welcome to contribute practically anything to Saucedacity! We can always use ideas for our next 1.3 and future releases. Of course, be sure to read our code of conduct and contributing guidelines before contributing.

I don't exactly know if this project is dead or not. Some might think so, from what I can tell, it might be on hiatus, etc. Nevertheless, I would also like to open up some opportunities to work together! I believe this will benefit both communities as a whole because we can share fixes, features, ideas, etc., among both. I do have some very large changes, such as a migration to Qt (with a UI redesign as a planned outcome), planned that are starting to get under way, so if you guys want a challenge with Qt and architectural restructuring, we might have a good challenge for you!

Nevertheless, if you guys don't want to contribute, that's okay! I hope we can collaborate at one point and exchange ideas, fixes, patches, all that good stuff. If you guys want to contribute to Saucedacity, then welcome! We have a variety of things ready for you! :)

So, that's all I have to say for now. Whether you guys consider Saucedacity or not, I wish you guys the best of luck on your journeys with whatever you guys have, including any adventures with Tenacity. I also hope to collaborate with you guys in the future too for the great or good.

Cheers! :)

**Comment by fitojb** [comment]
IMHO, bureaucracy killed this project

**Reply by FrostKnight** [reply]
Doesn't mean you have to surrender... someone else is still working on a fork, why not just join that dev?

I mean... why should the new devs of audacity win... at all.

Especially, given they plan to manipulate it, for who knows what...

Perhaps in the future, the original version will become completely spyware...

If it isn't already...

My point being, please join the other fork, the one that is being maintained and doesn't have a bunch of nutjobs and/or untrustworthy shmucks attached to it.

Again. I appeal to those who want an audacity like experience, but without untrustworthy people or nutcases attached, or both to join this project:

https://github.com/saucedacity/saucedacity

That being said,  you can ignore me, if you want, I don't have the skills to do this work, but giving up at this point makes less sense given a fork is still being worked on...

:)

If its too much work, I hear you, loud and clear!

But if its just because you think audacity can't be beat, then... I disagree entirely...

Just don't make a choice you will regret, whatever you do though.  Btw, when I said nutcases, I was definitely not including this project  or the one I linked to, and I definitely was not only including audacity.

I will leave it at that...

**Reply by poyribn** [reply]
i also don't have the skills to work on an audacity fork.
if i need an audio editor before Tenacity/Saucedacity is ready,
i might use my old copy of Audacity 2.x — that version was made before muse team took over,
or make a specialized audio editor for my needs from scratch.

**Comment by mc776** [comment]
I've been using a build of Tenacity from last August for everything since I downloaded it and everything works fine.

**Comment by generic-pers0n** [comment]
Well, guys, it looks like things are changing :)

**Reply by poyribn** [reply]
i have subscribed to notifications from the [𝘀aucedacity](https://github.com/tenacityteam/saucedacity/) repository.
𝘀aucedacity is a fork of the original 𝘁enacity repository. the original 𝘁enacity repository is now tenacity-legacy

i can confirm that 𝘀aucedacity is actively continued to be developed

𝘀aucedacity is also being rebranded to 𝘁enacity — with permission from the original 𝘁enacity developers
